### PR TITLE
.gitignore: add *.rej and *.save 

### PR DESCRIPTION
**Pull request by benpicco**
### Contribution description

The output of my `git status` has become quite messy. Let's ignore remnants of failed patches so the output is a bit cleaner.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,8 +21,12 @@ fuzzing/**/findings/
 .*.swn
 .*.swo
 .*.swp
+*.save
+*.rej
 \#*\#
 cachegrind.out*
+# Core dumps
+core
```
Comment (miri64) [inline .gitignore:29]: Given that we have a directory `core/` I don't know if this is a good idea to ignore...

**Review by miri64 — CHANGES_REQUESTED** [review]

**Comment by miri64** [comment]
Apart from the `core` thing, I think this can go in, but pro-tip: for local ignores you can also amend to `.git/info/exclude`, see also [gitignore(5)](https://git-scm.com/docs/gitignore).

File: .gitignore (hunk 1 above)
Comment (benpicco) [reply .gitignore:29]: Changes in `core/` are still displayed with this. I think for this to capture the core directory we would have to add `core/` here.

**Review by miri64 — APPROVED** [review]
ACK

**Pull request merged**